build.gradle.kts

```diff
@@ -28,7 +28,7 @@ plugins {
 // of band with the rest of the projects.
 allprojects {
     group = "software.amazon.smithy"
-    version = "0.8.1"
+    version = "0.9.0"
```

Comment:
Check the project for older versions of the packages in text (0.7.0, 0.8.0, 0.8.1, etc.) that might need to be updated as well. There are a few in `/docs/` for certain.